daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/transaction/VersionTimeline.scala

```diff
@@ -53,6 +53,7 @@ private[lf] object VersionTimeline {
       That(LanguageVersion(LMV.V1, "4")),
       That(LanguageVersion(LMV.V1, "5")),
       That(LanguageVersion(LMV.V1, Dev)),
+      This(That(TransactionVersion("8"))),
```

Comment:
Let's put the 8 above Dev; see the comment below that addresses why dev versions are weird.

Reply:
moved.